.github/workflows/build.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
       arch: "amd64"
       branch: ${{ inputs.branch }}
       build_type: ${{ inputs.build_type || 'branch' }}
-      container_image: "rapidsai/ci-conda:latest"
+      container_image: "rapidsai/ci-conda:25.08-latest"
```

Comment:
This change was made AFTER the `25.10` branch was created, so we'll need to re-run `ci/release/update-version.sh` so that these image tags are all `25.10` instead